NEWS.md

```diff
@@ -1,3 +1,51 @@
+# New in snapd 2.66:
+* AppArmor prompting (experimental): expand kernel support checks
+* AppArmor prompting (experimental): consolidate error messages and add error kinds
+* AppArmor prompting (experimental): grant /v2/snaps/{name} via snap-interfaces-requests-control
+* AppArmor prompting (experimental): add checks for duplicate pattern variants
+* Registry views (experimental): add handlers that commit (and cleanup) registry transactions
+* Registry views (experimental): add a snapctl fail command for rejecting registry transactions
+* Registry views (experimental): allow custodian snap to implement registry hooks to modify and save registry data
+* Registry views (experimental): compute view/plugs affected by modified path
+* Registry views (experimental): make registry transactions serialisable
+* Snap components: handle refreshing components to revisions that have been on the system before
+* Snap components: enable creating Ubuntu Core images that contain components
+* Snap components: handle refreshing components independently of snaps
+* Snap components: handle removing components when refreshing a snap that no longer defines them
+* Snap components: extend snapd Ubuntu Core installation API to allow for picking optional snaps and components to install
+* Snap components: extend kernel.yaml with "dynamic-modules", allowing kernel to define a location for kmods from component hooks
+* Desktop IDs: add desktop-file-ids plug attribute to desktop interface
```

Comment:
This is duplicated in line 38, maybe we should remove it here
`* desktop interface: add desktop-file-ids plug attr to desktop interface`

Reply:
fixed, removed it here.